src/main/java/at/hannibal2/skyhanni/features/garden/AnitaMedalProfit.kt

```diff
@@ -98,7 +99,16 @@ class AnitaMedalProfit {
         val profit = itemPrice - fullCost
         val format = NumberUtil.format(profit)
```

Comment:
maybe this could be renamed to be a little more descriptive and used down below in the hover tooltip